suite-native/transactions/src/components/TransactionDetail/TransactionDetailHeader.tsx

```diff
@@ -25,6 +25,15 @@ type TransactionDetailHeaderProps = {
 const ICON_SIZE = 56;
 const ICON_SPINNER_WIDTH = 3;
 
+const failedTxnTxtStyle = prepareNativeStyle<{ isFailedTx: boolean }>((_, { isFailedTx }) => ({
```

Comment:
Not sure what `TxnTxt` means, I'd go with just this:
```suggestion
const failedTxStyle = prepareNativeStyle<{ isFailedTx: boolean }>((_, { isFailedTx }) => ({
```

Same for other occurrences.

Reply:
d7733272f4